test/WebSites/RazorWebSite/Controllers/PartialViewEngineController.cs

```diff
@@ -42,5 +42,15 @@ public IActionResult PartialWithModel()
             };
             return PartialView(model);
         }
+
+        public IActionResult RenderPartialWithModel()
```

Comment:
All the functional tests run under an in-memory host. Running it should be identical to a unit test. We went through an exercise of trimming functional tests so it really doesn't make to add coverage for trivial scenarios.